### DISCUSSION TITLE
Feature request: Collapsible categories in control panel

### DISCUSSION THREAD

**Discussion by ttmwebdesign** [ExpressionEngine/ExpressionEngine#2917] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
In the categories page in the CP, click-and-dragging of nested categories can become virtually unmanageable the more child/grandchild categories a parent category has.

**Describe the solution you'd like**
I would really like to see child, child of child, etc., categories that collapse beneath their parents. Accordion style.

**Describe alternatives you've considered**
The alternative for now is making the browser window as long as possible. Potentially even requiring to change monitor orientation to portrait (so, longer than it is wide) to give more length.

Thanks!

(attached screenshot of a particularly gnarly category with four children, three of those who have two children of their own, and the last one has seven.)
<img width="907" alt="screenshot-client" src="https://user-images.githubusercontent.com/12754955/80239937-ea350580-8615-11ea-9190-b072b59e38f9.png">

**Comment by ttmwebdesign** [comment]
Just to add, the sample image isn't even the gnarliest category parent this client has... the longest one is a parent with seven children, two of which have seven and eight children of their own. I'm not sure whether that one can even be moved given how much page real estate it consumes, and none of the browsers seem to want to scroll down when I've got the parent clicked to drag. I am not even sure changing my monitor orientation on my second monitor would help in the case of that one, as its screen resolution isn't as good as my iMac (where the screenshot was taken from).

**Comment by Yulyaswan** [comment]
PR #3843
